dev/Projections/CS/Microsoft.Windows.Foundation/DecimalExtensions.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.CompilerServices;
+
+using Microsoft.Windows.Foundation;
+
+namespace Microsoft.Windows.Foundation
+{
+    public static class DecimalExtensions
+    {
+        /// Return a WinRT Decimal object.
+        public static Microsoft.Windows.Foundation.Decimal ToDecimal(this System.Decimal d)
```

Comment:
Should use the keyword in source, not the fully qualified type name, as it's a primitive in C# (same below)
```suggestion
        public static Microsoft.Windows.Foundation.Decimal ToDecimal(this decimal d)
```